applications/dashboard/openapi/tick.yml

```diff
@@ -0,0 +1,32 @@
+openapi: 3.0.2
+info:
+paths:
+  /tick:
+    post:
+      summary: Register a page view for statistical and analytical purpose.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TickRequest'
+        required: false
+      responses:
+        '201':
+          description: Success
+      tags:
+      - Statistics
+components:
+  schemas:
+    TickRequest:
+      description: Both body request parameters are optional
+      properties:
+        Path:
+          description: Page path
+          minLength: 1
+          type: string
+          example: /discussions/123
+        ResolvedPath:
```

Comment:
Can you leave this one out of the docs? It is kind of outdated and I'd rather new code doesn't depend on it.